lib/util/json-log-humanizer.js

```diff
@@ -53,6 +48,19 @@ humanizer.processLine = function(line) {
   }
 };
 
+function extraData(data) {
+  var extras = "";
+  if (data.timestamp) {
+    extras += " timestamp: " + data.timestamp;
```

Comment:
Also seems like the logging facility (winston or something like it) could do this for you.